packages/rocketchat-ldap/ldap_server.js

```diff
@@ -287,6 +290,51 @@ Accounts.registerLoginHandler("ldap", function(loginRequest) {
 			throw new Meteor.Error("LDAP-login-error", "LDAP Authentication succeded, but no user exists in Mongo. Either create a user for this email or set LDAP_DEFAULTS.createNewUser to true");
 		}
 
+		// LDAP sync data logic
+		var syncUserData = RocketChat.settings.get('LDAP_Sync_User_Data');
+		var syncUserDataFieldMap = RocketChat.settings.get('LDAP_Sync_User_Data_FieldMap').trim();
+		if (userId && syncUserData && syncUserDataFieldMap) {
+			var userData = {};
+			var fieldMap = JSON.parse(syncUserDataFieldMap);
+
+			var = emailList = [];
```

Comment:
wow, good catch. That's why I hate multitasking, tried to tackle this without giving proper attention :)